sycl/include/sycl/ext/oneapi/matrix/matrix-tensorcore.hpp

```diff
@@ -0,0 +1,274 @@
+#pragma once
+
+#include <CL/sycl/detail/defines_elementary.hpp>
+#include <immintrin.h>
+
+__SYCL_INLINE_NAMESPACE(cl) {
+namespace sycl {
+namespace ext {
+namespace intel {
```

Comment:
We changed recently the name space. oneapi instead of Intel.
Please refer to matrix-jit.hpp rather than matrix-aot.hpp as it has the newest syntax of the specification.

Reply:
Thanks